person.rb

```diff
@@ -1,15 +1,16 @@
 require './nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] It is required to have setter and getter methods for the `:rentals`. It could be simply added to the already written methods for name and age.

```
attr_accessor :name, :age, :rentals